control/control.MicroOS.xml

```diff
@@ -70,6 +70,12 @@ textdomain="control"
         <!-- bnc #431158: Adjusts /etc/sysconfig/security/POLKIT_DEFAULT_PRIVS if set -->
         <polkit_default_privs>restrictive</polkit_default_privs>
 
+        <!-- Set SELinux enforcing mode by default -->
+        <selinux>
+          <mode>enforcing</mode>
+          <configurable config:type="boolean">true</configurable>
+          <pattern>microos_selinux</pattern>
```

Comment:
Please add note about two selinux patterns also here. And especially part that why we pick this one so we can easily revisit in future.

Reply:
Done, see https://github.com/yast/skelcd-control-MicroOS/pull/30/commits/3888fae7e440fe84e7f67ce013e65535c645df5a